android/start.md

```diff
@@ -70,9 +77,11 @@ dependencies {
 }
 ```
 
-c. Sync the project and ensure it built successfully after the sync.
+**Note**
+If you get the following error message, "ERROR: Process 'command 'npx'' finished with non-zero exit value 1” this may be due to your user not having permissions to the node_modules folder on your machine. Follow the steps [at this link](https://docs.npmjs.com/resolving-eacces-permissions-errors-when-installing-packages-globally) to resolve it.
```

Comment:
1. Any reason we removed the instruction that says "Sync the project".  Adding the dependency and saving it may not build it for everyone as developers may have auto-sync/build turned off.
2. This error can happen only after the user builds a project using "Make Project", right ? If yes, this note should go in step c.

3. Updated wording: If you get the following error message: "ERROR: Process 'command 'npx'' finished with non-zero exit value 1”, this may be due to the logged in user on your machine having insufficient permissions to access the node_modules folder on your machine.

Reply:
1. Yes - Nikhil L said it's not necessary and I've tried it and it's not. Make project by itself is enough.

2. You're right this should now go after the new step C so I'll update that.

3. Ok